buildSrc/src/main/kotlin/com/islamversity/reyan/Dependencies.kt

```diff
@@ -209,6 +209,7 @@ object Deps {
         val test = "org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.coroutines}"
         val flowBinding = "ru.ldralighieri.corbind:corbind:${Versions.flowBinding}"
         val turbine = "app.cash.turbine:turbine:0.2.1"
+        val coroutinesWorker = "com.autodesk:coroutineworker:0.6.2"
```

Comment:
Coroutine guys had solved this issue [here](https://github.com/Kotlin/kotlinx.coroutines/issues/462#issuecomment-734844399)

[more information](https://github.com/Kotlin/kotlinx.coroutines/blob/native-mt/kotlin-native-sharing.md)